module4/assignment4.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Cole Marter | INFO 2439 | Assignment 4</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial scale=1">
+  <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css"/>
+<script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+<script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+<link rel="stylesheet" href="ctheme1/themes/assignment4.min.css" />
+<link rel="stylesheet" href="ctheme1/themes/jquery.mobile.icons.min.css" />
+<link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile.structure-1.4.5.min.css" />
+<link rel="stylesheet" href="_css/assignment4.css
+  </head>
+<body>
+<!-- Cole Marter
+     assignment4.htm
+     INFO2439
+     Thoendel
+     10/5/19
+-->
+<div data-role="page" id="page1" data-add-back-btn="true" data-title="Cole Marter | INFO 2439 | Assignment #4">
+  <div data-role="header" data-position="fixed">
+  <p>
+      <a href="" data-rel="back">Go back</a>
+      </p>
+      <h1>Bistro Customer Survey</h1>
+    </div>
+	<form action="http://emthoendel.faculty.mccinfo.net/2439/bistrosurvey_insert.php" method="post">
+	<fieldset data-role="controlgroup">
+ <legend>How Would You Rate Your Meal Today?</legend>
+<label for="meal_rate">Meal Rate</label>
+<select name="meal_rate" id="meal_rate">
+<option value="Poor">Poor</option>
+<option value="Needs Work">Needs Work</option>
+<option value="Fair">Fair</option>
+<option value="Good">Good</option>
+<option value="Excellent">Excellent</option>
+</select>
```

Comment:
the book shows using <div> tags for each label + input pair so that you can more easily maintain and focus enhancements.